dcos/dcos-ui
# DCOS-48339: volumes form section

Add the volumes section to the jobs form.

## Testing

- Open jobs form, go to Volumes tab
- Add volumes
- Check JSON editor to make sure they appear as expected
- Try to submit with invalid volumes (missing fields, wrong field type, etc)

## Trade-offs

N/A

## Dependencies

DCOS-48339 is not complete without EE changes.

## Screenshots

<img width="1353" alt="Screen Shot 2019-05-01 at 5 01 21 PM" src="https://user-images.githubusercontent.com/19582796/57050714-ca514880-6c32-11e9-952a-2daca8000603.png">
